core/src/price_estimation/clients/generic_client.rs

```diff
@@ -265,26 +291,16 @@ mod tests {
 
     #[test]
     fn get_token_prices_error() {
-        let mut api = MockDexagApi::new();
+        initialize_mockapi_context();
+        let mut api = MockApi::new();
 
         let tokens = hash_map! {
             TokenId(6) => TokenBaseInfo::new("DAI", 18, 0),
             TokenId(1) => TokenBaseInfo::new("ETH", 18, 0)
         };
 
         lazy_static! {
-            static ref API_TOKENS: [super::api::Token; 2] = [
-                super::api::Token {
-                    name: String::new(),
-                    symbol: "DAI".to_string(),
-                    address: None,
-                },
-                super::api::Token {
-                    name: String::new(),
-                    symbol: "ETH".to_string(),
-                    address: None,
-                },
-            ];
+            static ref API_TOKENS: [MockGenericToken; 2] = ["DAI".into(), "ETH".into(),];
```

Comment:
trailing comma?

Reply:
I thought rustfmt would take care of it, but this is not the case!
https://github.com/rust-lang/rustfmt/issues/1034